server/portal/apps/workspace/api/views.py

```diff
@@ -102,39 +101,24 @@ def get(self, request, *args, **kwargs):
 @method_decorator(login_required, name='dispatch')
 class JobsView(BaseApiView):
     def get(self, request, *args, **kwargs):
-        agave = request.user.tapis_oauth.client
-        job_id = request.GET.get('job_id')
+        tapis = request.user.tapis_oauth.client
+        job_uuid = request.GET.get('job_uuid')
 
         # get specific job info
-        if job_id:
-            data = agave.jobs.get(jobId=job_id)
-            q = {"associationIds": job_id}
-            job_meta = agave.meta.listMetadata(q=json.dumps(q))
-            data['_embedded'] = {"metadata": job_meta}
-
-            # TODO: Decouple this from front end somehow
-            archiveSystem = data.get('archiveSystem', None)
-            if archiveSystem:
-                archive_system_path = '{}/{}'.format(archiveSystem, data['archivePath'])
-                data['archiveUrl'] = '/workbench/data-depot/'
-                data['archiveUrl'] += 'agave/{}/'.format(archive_system_path.strip('/'))
-
-                jupyter_url = get_jupyter_url(
-                    archiveSystem,
-                    "/" + data['archivePath'],
-                    request.user.username,
-                    is_dir=True
-                )
-                if jupyter_url:
-                    data['jupyterUrl'] = jupyter_url
+        if job_uuid:
+            data = tapis.jobs.getJob(jobUuid=job_uuid)
 
         # list jobs
         else:
             limit = int(request.GET.get('limit', 10))
             offset = int(request.GET.get('offset', 0))
             period = request.GET.get('period', 'all')
```

Comment:
Interesting. I don't think `period` is configured on the frontend

Reply:
Ah yes.. I noticed that this wasn't used at all in the portals.
I guess it was open as a placeholder for future implementation? Is there a ticket for it?

Reply:
yeah, from cepv1. there one could click on like "last month" or something similar

Reply:
could we just drop that period as no longer used?

Reply:
@rstijerina any thoughts on this? I've just commented this for now (alongside the potential portal tag search implementation).

Reply:
@duckonomy I would remove it outright instead of commenting